### ISSUE TITLE
Disabling `vet` doesn't work

### ISSUE THREAD

**Issue by cmacrae** [alecthomas/gometalinter#434]
Yo @alecthomas 👋   

Thanks for `gometalinter`!  
I use it in conjunction with the FlyCheck package for Emacs.  

I'm running into a weird issue where, even with `--disable=vet` it's complaining that `govet` isn't found in my `PATH`:  
```
<?xml version="1.0" encoding="UTF-8"?>
<checkstyle version="5.0"></checkstyle>
WARNING: exec: "govet": executable file not found in $PATH
```
This is the case for v2 installed from gopkg.in or directly from `HEAD`.  

Any idea why `--disable=vet` doesn't seem to be working?
Also as I understand it, since Go 1.9 - the issue for which this `govet` fork was created should now be resolved: https://github.com/golang/go/issues/22296  

Thanks in advance! Please let me know if there's any further information I can provide

**Comment by dnephin** [comment]
I think it might be due to `vetshadow` not being disabled. Try disabling that as well.

> as I understand it, since Go 1.9 - the issue for which this govet fork was created should now be resolved: golang/go#22296

`gometalinter` can still be used with older go versions, but I think we could add a check to use the canonical version if the go version is >= 1.9, and use the forked version for older go.

**Comment by cmacrae** [comment]
Hey @dnephin - yep, you are correct. I also disabled `vetshadow` and no longer see the problem.  
Yeah, I agree - I think that's a good approach; falling back to the forked vet if the version is older than 1.9.  

Nice one for opening the above 👍   I'll close this off.
